test/interpret.jl

```diff
@@ -156,3 +156,19 @@ ex = quote
 end
 frame = JuliaInterpreter.prepare_toplevel(Main, ex)
 @test JuliaInterpreter.finish_and_return!(JuliaStackFrame[], frame, true) == 1
+
+
+# "correct" line numbers
+defline = @__LINE__() + 1
+function f(x)
+    x = 2x
+    # comment
+    # comment
+    x = 2x
+    # comment
+    return x*x
+end
+frame = JuliaInterpreter.enter_call_expr(:($(f)(3)))
```

Comment:
Not important to change, but FYI you can also use `enter_call(f, 3)`.

Reply:
Neat, changed to that.